From bbeare1@gmail.com  Fri Dec 18 10:08:03 2009
From: Bruce Beare <bbeare1@gmail.com>
Date: Fri, 20 Nov 2009 15:46:11 -0800
Subject: Staging: comedi: pcmad: [PATCH 3/3] cleanup printk() warnings
To: gregkh@suse.de
Cc: devel@driverdev.osuosl.org, Bruce Beare <bbeare1@gmail.com>, linux-kernel@vger.kernel.org
Message-ID: <1258760771-24311-4-git-send-email-bbeare1@gmail.com>

cleanup printk() warnings for including the facility level.
cleanup add a printk("\n") to terminate the print for the non-error case.


Signed-off-by: Bruce Beare <bbeare1@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/comedi/drivers/pcmad.c |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

--- a/drivers/staging/comedi/drivers/pcmad.c
+++ b/drivers/staging/comedi/drivers/pcmad.c
@@ -134,11 +134,12 @@ static int pcmad_attach(struct comedi_de
 	unsigned long iobase;
 
 	iobase = it->options[0];
-	printk("comedi%d: pcmad: 0x%04lx ", dev->minor, iobase);
+	printk(KERN_NOTICE "comedi%d: pcmad: 0x%04lx ", dev->minor, iobase);
 	if (!request_region(iobase, PCMAD_SIZE, "pcmad")) {
-		printk("I/O port conflict\n");
+		printk(KERN_CONT "I/O port conflict\n");
 		return -EIO;
 	}
+	printk(KERN_CONT "\n");
 	dev->iobase = iobase;
 
 	ret = alloc_subdevices(dev, 1);
@@ -165,7 +166,7 @@ static int pcmad_attach(struct comedi_de
 
 static int pcmad_detach(struct comedi_device *dev)
 {
-	printk("comedi%d: pcmad: remove\n", dev->minor);
+	printk(KERN_NOTICE "comedi%d: pcmad: remove\n", dev->minor);
 
 	if (dev->irq)
 		free_irq(dev->irq, dev);
